package-lock.json

```diff
@@ -33674,6 +33674,21 @@
         "tslib": "^2.6.2"
       }
     },
+    "packages/host-inventory/node_modules/axios": {
```

Comment:
@apinkert I think this is leftover from our host-inventory commits

Reply:
reverted